backend/curriculum_tracking/tests/test_AgileCard.py

```diff
@@ -688,3 +687,128 @@ def test_cancel_workshop_attendance(self):
+        self.user = self.card_1.assignees.first()
+        self.assertIsNotNone(self.card_1.assignees)
+        self.assertIsNotNone(self.card_2.assignees)
+        self.assertIsNotNone(self.card_1.recruit_project)
+        self.assertIsNotNone(self.card_2.recruit_project)
+        self.assertEqual(self.card_1.status, AgileCard.IN_PROGRESS)
+        self.assertEqual(self.card_2.status, AgileCard.IN_PROGRESS)
+        self.assertEqual(self.project_1.content_item, self.card_1.content_item)
+        self.assertEqual(self.project_2.content_item, self.card_2.content_item)
+
+        # Starting the projects (project_1 & project_2) and asserting that they are in status IN_PROGRESS
+        self.project_1.start_time = timezone.now() - timedelta(days=5)
+        self.project_2.start_time = timezone.now() - timedelta(days=5)
+        self.project_1.save()
+        self.project_2.save()
+        self.assertEqual(AgileCard.derive_status_from_project(self.project_1), AgileCard.IN_PROGRESS)
+        self.assertEqual(AgileCard.derive_status_from_project(self.project_2), AgileCard.IN_PROGRESS)
+
+        # Setting a request for review and creating four different review times (No reviews done yet)
+        self.request_review_time = self.project_1.start_time + timedelta(1)
+        self.project_1.request_review(force_timestamp=self.request_review_time)
+        self.time_one = self.project_1.start_time - timedelta(days=6)
+        self.time_two = self.project_1.start_time + timedelta(days=4)
+        self.time_three = self.project_1.start_time + timedelta(days=3)
+        self.time_four = self.project_1.start_time + timedelta(days=2)
+
+    def test_functionality_of_users_that_reviewed_since_last_review_request_for_project_1(self):
+
+        # Four reviews are made with the four review times above (No reviews done on project_2)
+        review_1 = factories.RecruitProjectReviewFactory(
+            status=NOT_YET_COMPETENT,
+            recruit_project=self.project_1,
+            timestamp=self.time_one
+        )
+        review_1.timestamp = self.time_one
+        review_1.save()
+
+        review_2 = factories.RecruitProjectReviewFactory(
+            status=COMPETENT,
+            recruit_project=self.project_1,
+        )
+        review_2.timestamp = self.time_two
+        review_2.save()
+
+        review_3 = factories.RecruitProjectReviewFactory(
+            status=EXCELLENT,
+            recruit_project=self.project_1,
+        )
+        review_3.timestamp = self.time_three
+        review_3.save()
+
+        review_4 = factories.RecruitProjectReviewFactory(
+            status=NOT_YET_COMPETENT,
+            recruit_project=self.project_1,
+        )
+        review_4.timestamp = self.time_four
+        review_4.save()
+
+        project_1_return_results = self.card_1.get_users_that_reviewed_since_last_review_request()
+        ids_which_can_be_returned = [review_2.reviewer_user.id, review_3.reviewer_user.id, review_4.reviewer_user.id]
```

Comment:
a better name would be `ids_that_should_be_returned` or `expected_ids`

If something `can` be returned then it sounds like it's ok if it's not returned. Eg: "you can pay your taxes" makes it seem like you dont really have to.